src/aleph/handlers/content/vm.py

```diff
@@ -289,6 +301,50 @@ class HasParent(Protocol):
             )
 
 
+def get_extra_storage(content: InstanceContent, session: DbSession) -> int:
```

Comment:
Rename to `get_volume_size`? Irrelevant here, but this function could also be called on `ProgramContent` objects. Reflect that in the type hints.